src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.service;
+
+import java.util.List;
+import mate.academy.model.CinemaHall;
+
+public interface CinemaHallService {
+    CinemaHall add(CinemaHall movie);
```

Comment:
The parameter name for the `add` method should reflect that it's a CinemaHall being added, not a movie. Consider renaming the parameter to `cinemaHall` to improve code clarity.